### PR TITLE
FIX: PEP8 in denoise

### PR DESCRIPTION
**Pull request by manu-tej**
issues #875 fixed

### CODE REVIEW

File: dipy/denoise/tests/test_noise_estimate.py [hunk 1]
```diff
@@ -49,53 +54,68 @@ def test_piesno():
...
-    sigma = estimate_sigma(5 * np.ones((7, 7, 7, 3)), disable_background_masking=False)
+    sigma = estimate_sigma(5 * np.ones((7, 7, 7, 3)),
+                           disable_background_masking=False)
     assert_equal(sigma, np.array([0., 0., 0.]))
 
     arr = np.zeros((3, 3, 3))
     arr[0, 0, 0] = 1
     sigma = estimate_sigma(arr, disable_background_masking=False, N=1)
-    assert_array_almost_equal(sigma, 0.10286889997472792 / np.sqrt(0.42920367320510366))
+<<<<<<< HEAD
```
Comment (arokem) [inline dipy/denoise/tests/test_noise_estimate.py]: These conflict markers are going to cause some trouble... You might want to run the tests on your machine before you push the code up. You can do that by running `nosetests` in your source-code repository (assuming you've set up you paths correctly).

Comment (manu-tej) [inline dipy/denoise/tests/test_noise_estimate.py]: I would like to know more about running test on local machine.

Comment (arokem) [inline dipy/denoise/tests/test_noise_estimate.py]: We use nose for testing: https://nose.readthedocs.org/en/latest/

For a gentle introduction to this approach, take a look here:
http://katyhuff.github.io/python-testing/

There are videos with a tutorial starting here:

https://www.youtube.com/watch?v=T0BE9ApIegc

File: dipy/denoise/tests/test_noise_estimate.py [hunk 2]
```diff
@@ -110,6 +125,10 @@ def test_estimate_sigma():
 
     arr[0, 0, 0] = 1
     sigma = estimate_sigma(arr, disable_background_masking=True, N=12)
-    assert_array_almost_equal(sigma, np.array([0.46291005 / np.sqrt(0.4946862482541263),
-                                               0.46291005 / np.sqrt(0.4946862482541263),
-                                               0.46291005 / np.sqrt(0.4946862482541263)]))
+    assert_array_almost_equal(sigma,
```
Comment (samuelstjean) [inline dipy/denoise/tests/test_noise_estimate.py]: This is hard to read, no need to break lines for the 80 characters rule everytime if it does not really help.

Comment (manu-tej) [inline dipy/denoise/tests/test_noise_estimate.py]: Should I change it to 
`assert_array_almost_equal(sigma, np.array([0.46291005 / np.sqrt(0.4946862482541263),
                                                                         0.46291005 / np.sqrt(0.4946862482541263),
                                                                         0.46291005 / np.sqrt(0.4946862482541263)]))`

File: dipy/denoise/tests/test_noise_estimate.py [hunk 3]
```diff
@@ -49,53 +54,63 @@ def test_piesno():
...
-    assert_array_almost_equal(sigma, np.array([0.10286889997472792 / np.sqrt(0.42920367320510366),
-                                               0.10286889997472792 / np.sqrt(0.42920367320510366),
-                                               0.10286889997472792 / np.sqrt(0.42920367320510366)]))
+    assert_array_almost_equal(sigma,
+                              np.array([0.10286889997472792 /
+                                        np.sqrt(0.42920367320510366),
+                                        0.10286889997472792 /
+                                        np.sqrt(0.42920367320510366),
+                                        0.10286889997472792 /
+                                        np.sqrt(0.42920367320510366)]))
```
Comment (manu-tej) [inline dipy/denoise/tests/test_noise_estimate.py]: @samuelstjean Should I even change here?

File: dipy/denoise/noise_estimate.py [hunk 4]
```diff
@@ -295,9 +302,9 @@ def estimate_sigma(arr, disable_background_masking=False, N=0):
     (see [1]_, equation 18) with theta = 0.
     Since this function was introduced in [2]_ for T1 imaging,
     it is expected to perform ok on diffusion MRI data, but might oversmooth
-    some regions and leave others un-denoised for spatially varying noise profiles.
-    Consider using :func:`piesno` to estimate sigma instead if visual inacuracies
-    are apparent in the denoised result.
+    some regions and leave others un-denoised for spatially varying noise
+    profiles. Consider using :func:`piesno` to estimate sigma instead if visual
+    inacuracies are apparent in the denoised result.
```
Comment (arokem) [inline dipy/denoise/noise_estimate.py]: Typo (was there before, but maybe we fix it here?): 

"inacuracies"=> "inaccuracies"

**Comment by arokem** [comment]
Looks good to me. If you want to fix that typo, please do. Otherwise, if no one objects, I can merge this in a couple of days.

**Comment by manu-tej** [comment]
@arokem fixed the typo

**Comment by arokem** [comment]
Thanks. This is ready to go from my point of view.

**Comment by arokem** [comment]
closes #875

**Pull request merged**